packages/async-flow/test/replay-membrane-eventual.test.js

```diff
@@ -215,3 +223,22 @@ test.serial('test durable replay-membrane settlement', async t => {
   const zone3 = makeDurableZone(getBaggage(), 'durableRoot');
   return testReplay(t, zone3);
 });
+
+test.serial('test durable disabled eventual send', async t => {
+  annihilate();
+
+  nextLife();
+  const zone1 = makeDurableZone(getBaggage(), 'durableRoot');
+  await testFirstPlay(t, zone1);
```

Comment:
Why are we doing the first play in normal mode?

Reply:
I was just lazy in updating both async functions.  Both are now tried without and with eventual send support.